### PR TITLE
config: qualify the name of the version field

### PR DESCRIPTION
**Pull request by vbatts**
https://github.com/opencontainers/specs/issues/110

Signed-off-by: Vincent Batts <vbatts@hashbangbash.com>

### CODE REVIEW

**Comment by vbatts** [comment]
Closes #110

**Comment by tianon** [comment]
LGTM

**Comment by vishh** [comment]
LGTM

File: config.go [hunk 1]
```diff
@@ -5,7 +5,7 @@ package specs
 // bundle is packaged for distribution.
 type Spec struct {
 	// Version is the version of the specification that is supported.
-	Version string `json:"version"`
+	Version string `json:"spec_version"`
```
Comment (wking) [inline config.go]: I'd [rather have a more specific name][1].  Presumably the other versioned things were distinguishing from also have specs (and the current JSON pattern is camelCase, not underscores).

[1]: https://github.com/opencontainers/specs/issues/110#issuecomment-171470005

File: config.md [hunk 2]
```diff
@@ -8,14 +8,17 @@ This includes the process to run, environment variables to inject, sandboxing fe
 
 Below is a detailed description of each field defined in the configuration format.
 
-## Manifest version
+## Specification version
 
-* **`version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OCF specification with which the container bundle complies. The Open Container spec follows semantic versioning and retains forward and backward compatibility within major versions. For example, if an implementation is compliant with version 1.0.1 of the spec, it is compatible with the complete 1.x series. NOTE that there is no guarantee for forward or backward compatibility for version 0.x.
+* **`spec_version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OpenContainer specification with which the bundle complies.
```
Comment (mrunalp) [inline config.md]: should be specVersion

Comment (vbatts) [inline config.md]: i already pushed a change for this. PTAL

Comment (wking) [inline config.md]: On Thu, Jan 14, 2016 at 02:37:10PM -0800, Vincent Batts wrote:
> > +* **`spec_version`** (string, required) must be in [SemVer v2.0.0](http://semver.org/spec/v2.0.0.html) format and specifies the version of the OpenContainer specification with which the bundle complies.
> 
> i already pushed a change for this. PTAL

4df6c3bb (the current tip) has specVersion in config.go but
spec_version in config.md (twice).  Are we sure we don't want a more
specific name [1,2]?

[1]: https://github.com/opencontainers/specs/pull/309#discussion_r49667293
[2]: https://github.com/opencontainers/specs/issues/110#issuecomment-171470005

Comment (jonboulle) [inline config.md]: +1 to more specific name.

Comment (duglin) [inline config.md]: `ociVersion`

Comment (mrunalp) [inline config.md]: +1 to ociVersion

**Comment by vbatts** [comment]
updated to `ociVersion`. PTAL

**Comment by duglin** [comment]
+1

**Comment by tianon** [comment]
re-LGTM

**Comment by wking** [comment]
ociVersion is better, but we may end up with multiple OCI specs.
We're calling this spec “Open Container Specifications” [1], so why
not “ocsVersion”?

[1]: https://github.com/opencontainers/specs/tree/v0.2.0#open-container-specifications

**Comment by duglin** [comment]
Not to take us down too much of a tangent, but do we have one spec or multiple in this project? If we have multiple then "ocs" doesn't help since having "s=spec" when there are multiple is kind of ambiguous. Perhaps "Open Container Runtime (ocr)" for the core spec we've been talking about.   But if we view this all as one big spec (which might be nice to avoid each one having a different version number), then just "oci" would work.

**Comment by wking** [comment]
On Fri, Jan 15, 2016 at 10:35:34AM -0800, Doug Davis wrote:
> Not to take us down too much of a tangent, but do we have one spec
> or multiple in this project?

I would love to have this question answered, since I'm frequently
suggesting we handle orthogonal issues in separate OCI specs
(e.g. [1]).  But to stay on topic here, maybe we should move this
discussion to the list?

[1]: https://groups.google.com/a/opencontainers.org/d/msg/dev/uo11avcWlQQ/Tn0nFymREAAJ
     Subject: Re: appc + oci harmonization progress
     Date: Thu, 13 Aug 2015 11:29:08 -0700
     Message-ID: <20150813182908.GS412@odin.tremily.us>

**Comment by hqhq** [comment]
We should leave the discussion about the needs of multiple OCI spec to the list.
LGTM

**Pull request merged**